docs/release-notes/.FSharp.Core/8.0.300.md

```diff
@@ -6,3 +6,4 @@
 ### Fixed
 
 * Preserve original stack traces in resumable state machines generated code if available. ([PR #16568](https://github.com/dotnet/fsharp/pull/16568))
+* Enforce AttributeTargets on structs and classes ([PR #16790](https://github.com/dotnet/fsharp/pull/16790))
```

Comment:
I think this should say that we extended `RequireQualifiedAccessAttribute` and `AutoOpenAttribute` to work on structs as well.

Reply:
Done